org.restlet.java/org.restlet.ext.jetty/src/main/java/org/restlet/ext/jetty/JettyServerHelper.java

```diff
@@ -209,616 +209,570 @@
+		} else {
+			result = Long.parseLong(shutdownTimeoutValue);
+		}
+
+		return result;
+	}
+
+	/**
+	 * Thread pool idle timeout in milliseconds. Defaults to 60000.
+	 * <p>
+	 * Threads that are idle for longer than this period may be stopped.
+	 * 
+	 * @return Thread pool idle timeout.
+	 */
+	public int getThreadPoolIdleTimeout() {
+		return Integer.parseInt(getHelpedParameters().getFirstValue("threadPool.idleTimeout", "60000"));
+	}
+
+	/**
+	 * Thread pool maximum threads. Defaults to 200.
+	 * 
+	 * @return Thread pool maximum threads.
+	 */
+	public int getThreadPoolMaxThreads() {
+		return Integer.parseInt(getHelpedParameters().getFirstValue("threadPool.maxThreads", "200"));
+	}
+
+	/**
+	 * Thread pool minimum threads. Defaults to 8.
+	 * 
+	 * @return Thread pool minimum threads.
+	 */
+	public int getThreadPoolMinThreads() {
+		return Integer.parseInt(getHelpedParameters().getFirstValue("threadPool.minThreads", "8"));
+	}
+
+	/**
+	 * Thread pool stop timeout in milliseconds. Defaults to 5000.
+	 * <p>
+	 * The maximum time allowed for the service to shut down.
+	 * 
+	 * @return Thread pool stop timeout.
+	 */
+	public long getThreadPoolStopTimeout() {
+		return Long.parseLong(getHelpedParameters().getFirstValue("threadPool.stopTimeout", "5000"));
+	}
+
+	/**
+	 * Thread pool threads priority. Defaults to {@link Thread#NORM_PRIORITY}.
+	 * 
+	 * @return Thread pool maximum threads.
+	 */
+	public int getThreadPoolThreadsPriority() {
+		return Integer.parseInt(getHelpedParameters().getFirstValue("threadPool.threadsPriority",
+				String.valueOf(Thread.NORM_PRIORITY)));
+	}
+
+	/**
+	 * Returns the wrapped Jetty server.
+	 * 
+	 * @return The wrapped Jetty server.
+	 */
+	protected org.eclipse.jetty.server.Server getWrappedServer() {
+		if (this.wrappedServer == null) {
+			this.wrappedServer = createServer();
+		}
+		return this.wrappedServer;
+	}
+
+	/**
+	 * Sets the wrapped Jetty server.
+	 * 
+	 * @param wrappedServer The wrapped Jetty server.
+	 */
+	protected void setWrappedServer(org.eclipse.jetty.server.Server wrappedServer) {
+		Objects.requireNonNull(wrappedServer);
+		this.wrappedServer = wrappedServer;
+	}
+
+	@Override
+	public void start() throws Exception {
+		super.start();
+		org.eclipse.jetty.server.Server server = getWrappedServer();
+		ServerConnector connector = (ServerConnector) server.getConnectors()[0];
+
+		getLogger().info("Starting the Jetty " + getProtocols() + " server on port " + getHelped().getPort());
+		try {
+			server.start();
+			// We won't know the local port until after the server starts
+			setEphemeralPort(connector.getLocalPort());
+		} catch (Exception e) {
+			// Make sure that all resources are released, otherwise thread-pool
+			// may still be running.
+			server.stop();
+			throw e;
+		}
+
+	}
+
+	@Override
+	public void stop() throws Exception {
+		getLogger().info("Stopping the Jetty " + getProtocols() + " server on port " + getHelped().getPort());
```

Comment:
hi @jlouvel I see that the number of spaces for a tab is 8 with my Eclipse installation.
Do you see the same?